doc/classes/JavaObject.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="JavaObject" inherits="RefCounted" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
+	<brief_description>
+		Represents an object from the Java Native Interface.
+	</brief_description>
+	<description>
+		Represents an object from the Java Native Interface. It can be return from Java methods called on [JavaClass]. See [JavaClassWrapper] for an example.
+		[b]Note:[/b] This class only works on Android. For any other build, this class does nothing.
```

Comment:
Saying "platform" is a lot more common (I know it's not your wording, it's mine)
```suggestion
		[b]Note:[/b] This class only works on Android. On any other platform, this class does nothing.
```